.travis.yml

```diff
@@ -10,10 +10,11 @@ cache:
     - $HOME/.m2
 jdk:
   - oraclejdk8
-script: ./gradlew test testSRA testIntelDeflater
+script: ./gradlew jacocoTestReport testSRA testIntelDeflater;
 after_success:
   - echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'";
     echo "JAVA_HOME='$JAVA_HOME'";
+    ./gradlew coveralls;
     if [ "$TRAVIS_BRANCH" == "master" ]; then
        ./gradlew uploadArchives;
```

Comment:
I am just curious, will this upload archives in a fork of htsjdk? or if this is a pull request?  Something like this might work:
```
test "$TRAVIS_REPO_SLUG" == 'samtools/htsjdk' && test "$TRAVIS_PULL_REQUEST" == 'false' && test "$TRAVIS_BRANCH" == 'master'
```